rpc/types/utils.go

```diff
@@ -125,7 +125,7 @@ func EthTransactionsFromTendermint(clientCtx clientcontext.CLIContext, txs []tmt
 			continue
 		}
 		// TODO: Remove gas usage calculation if saving gasUsed per block
-		gasUsed.Add(gasUsed, ethTx.Fee())
+		gasUsed.Add(gasUsed, big.NewInt(int64(ethTx.GetGas())))
```

Comment:
GetGas() just gives the gas limit correct?
would it be better to use:
`gasUsed.Add(gasUsed, ethTx.Fee().Int64())`

Reply:
I think here need return gas used, not gas fee used, also, the number of Fee() is too big to return to the interface of web3.js, that will be crashed in web3.js lib cause too large number transform

Reply:
ethTx.Fee() return gaslimit * gasPrice, its much larger than real number of gas costed